client-examples/BrutalmazeClient.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Sockets;
+using System.Diagnostics;
+
+namespace BrutalmazeClient
+{
+	class Program
+	{
+		private static Socket ConnectSocket(string host, int port)
+		{
+			IPHostEntry ipHostInfo = Dns.GetHostEntry(host);
+			IPAddress ipAdress = ipHostInfo.AddressList[0];
+			IPEndPoint remoteEP = new IPEndPoint(ipAdress, port);
+			Socket tmp = new Socket(ipAdress.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
+			tmp.Connect(remoteEP);
+			return tmp;
+		}
+
+		static void Main(string[] args)
+		{
+			Process.Start("CMD.exe", "/C brutalmaze --server");
+			System.Threading.Thread.Sleep(1000);
+			const string host = "localhost";
+			const int port = 8089;
+			Socket client_socket = ConnectSocket(host, port);
+			Random rnd = new Random();
+
+			byte[] buff = new byte[1000];
+			byte[] query;
+			string[] matrix = new string[100];
+			const int MAGIC = 42; // For escape
+			int prevX = 1234, prevY = 5678;
+			int dir = 0, deg = 0, atk = 1;
+			int needEsc = 0;
+
+			while (42 < 420)
+			{
+				int recv, sent;
+				try
+				{ 
+					recv = client_socket.Receive(buff, 7, 0);
+				} catch (SocketException e)
+				{
+					break;
+				}
+				string l = Encoding.ASCII.GetString(buff, 0, 7);
+				int sz = Int32.Parse(l);
+				if (sz == 0)
+					break;
+				recv = client_socket.Receive(buff, sz, 0);
+				string data = Encoding.ASCII.GetString(buff, 0, sz);
+				// Standardize Data
+				int nl = 0;
+				string l1 = data.Split('\n')[nl];
+				int nh = Int32.Parse(l1.Split(' ')[0]);
+				int ne = Int32.Parse(l1.Split(' ')[1]);
+				int nb = Int32.Parse(l1.Split(' ')[2]);
+				int score = Int32.Parse(l1.Split(' ')[3]);
+				for (int i = 0; i < nh; ++i, ++nl)
+				{
+					matrix[i] = data.Split('\n')[i + 1];
+				}
+				l1 = data.Split('\n')[++nl];
+				char hC = Char.Parse(l1.Split(' ')[0]);
+				int hX = Int32.Parse(l1.Split(' ')[1]);
+				int hY = Int32.Parse(l1.Split(' ')[2]);
+				int hA = Int32.Parse(l1.Split(' ')[3]);
+				int canAtk = Int32.Parse(l1.Split(' ')[4]);
+				int canReg = Int32.Parse(l1.Split(' ')[5]);
+				for(int i = 1; i <= ne; ++i, ++nl)
+				{
+				}
+				for(int i = 1; i <= nb; ++i, ++nl)
+				{
+				}
+				// Process
+
+				// Or just random 'cause it's really OP
```

Comment:
Algorithm doesn't matter for an example/template so whatever you use is not a problem, though I'm curious what *OP* stands for.